source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -68,24 +70,37 @@ def getCommentInfoFromPosition(position):
 		UIAElement=UIAElementArray.getElement(index)
 		UIAElement=UIAElement.buildUpdatedCache(UIAHandler.handler.baseCacheRequest)
 		obj=UIA(UIAElement=UIAElement)
-		if not obj.parent or obj.parent.name!='Comment':
+		if not obj.parent or (
+			obj.parent.role != controlTypes.ROLE_GROUPING and not

```

Comment:
I thought Flake8 checked this, apparently not in this situation: please put the logical operators at the start of the line, it's easy to miss them when they are at the end.

Reply:
Should I move "or (" from line 73 or only "and not" from line 74?

Reply:
Just the "and not"